### ISSUE TITLE
Bike power in .tcx

### ISSUE THREAD

**Issue by ventoux84** [Maproom/qmapshack#566]
### Is your feature request related to a problem? Please describe.
[comment]: # (A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] )My .tcx files from my smartphone contain values from Ant+ sensors like "Bike Power".
When I open a .tcx track in Qmapshack, I don't see this information.
It does not appear in the summary, nor in the possible graphs of this track.

### Describe the solution you'd like
[comment]: # (A clear and concise description of what you want to happen.)
I would like see it in the summary of the track, and be able to display the graph.

### Describe alternatives you've considered
[comment]: # (A clear and concise description of any alternative solutions or features you've considered.)
I certainly forget to set a parameter, but which one?

### Give a reason why someone else should spend time on that request
[comment]: # (Keep in mind that you ask others to spend their spare time for your request. Good reasons are if you are able to prove that the request solves a fundamental problem for everyone, or the request is of general interest for many users. Bad reasons are you just want it to have that way, or argue that software xyz does it.)
It is useful to analyze the progress of an outing

### Additional context
[comment]: # (Add any other context or screenshots about the feature request here.)
```
<Trackpoint>
<Time>2022-11-13T14:05:03.000Z</Time>
<Position>
<LatitudeDegrees>xx.0484195</LatitudeDegrees>
<LongitudeDegrees>x.1268042</LongitudeDegrees>
</Position>
<AltitudeMeters>146.54</AltitudeMeters>
<Cadence>50</Cadence>
<Extensions><ns3:TPX>
<ns3:Watts>27.26</ns3:Watts>
<ns3:Speed>6.31</ns3:Speed>
</ns3:TPX></Extensions>
</Trackpoint>
```

**Comment by wthaem** [comment]
Did you check comboboxes in the "Graphs" tab of the track edit window? One of the combobox options should be "Cadence" (if your TCX file has the correct data structure). Selecting this option you get your graph.
![cadence](https://user-images.githubusercontent.com/55309218/201539476-3d7a9b86-31e8-4750-82cf-a15832cbfe92.jpg)

**Comment by wthaem** [comment]
Did you check comboboxes in the "Graphs" tab of the track edit window? One of the combobox options should be "Cadence" (if your TCX file has the correct data structure). Selecting this option you get your graph.
![cadence](https://user-images.githubusercontent.com/55309218/201539476-3d7a9b86-31e8-4750-82cf-a15832cbfe92.jpg)

**Comment by ventoux84** [comment]
Yes, i can see the graph "cadence", and also the speed. I shouldn't have mentioned those
it is the one of power that is missing.

**Comment by runrun1234** [comment]
Hello guys,
At that time when import/export TCX function I just had an HR belt and no power sensor then I dropped power !
But this is not difficult to add it to https://github.com/Maproom/qmapshack/blob/dev/src/qmapshack/gis/tcx/CTcxProject.cpp.
A very good opportunity to step into QMapShack project !

**Comment by kkarsten62** [comment]
Just for your Information:

I am using for cycling Garmin Edge 1000 with powermeter pedals from Favero Assioma. All works fine out of the box with Garmin FIT file format and QMS.

![grafik](https://user-images.githubusercontent.com/55464608/202862764-f7f7b4d6-3795-4db7-b341-5d0d3bd52df8.png)
